common/app/common/RequestLogger.scala

```diff
@@ -37,7 +37,11 @@ case class RequestLoggerFields(request: RequestHeader, response: Option[Result],
     val guardianSpecificHeaders = allHeadersFields.view.filterKeys(_.toUpperCase.startsWith("X-GU-")).toMap
 
     (allowListedHeaders ++ guardianSpecificHeaders).toList.map { case (headerName, headerValue) =>
-      LogFieldString(s"req.header.$headerName", headerValue)
+      if (headerName == "x-gu-xid") {
+        LogFieldString(s"fastlyRequestId", headerValue)
+      } else {
+        LogFieldString(s"req.header.$headerName", headerValue)
+      }
```

Comment:
We were already logging the value of `x-gu-xid` with a log feild of `req.header.x-gu-xid`. But we want to have a common log field between frontend and DCR, so this field is renamed into `fastlyRequestId`